django-testbed/frontend/src/useEndpoint.js

```diff
@@ -9,8 +9,8 @@ import useSWR from 'swr';
  * @return Object Same values as returned by useSWR with the addition of `execute` which
  * can be used to execute the action directly, optionally with new arguments.
  */
-export default function useEndpoint(action, args, config) {
-    const paginator = usePaginator(action && action.getPaginatorClass());
+export default function useEndpoint(action, args, { paginationStatePair, ...config } = {}) {
```

Comment:
I was wondering about the consistency of `paginationStatePair` vs `paginationState` vs `paginationUseState` etc. [Here](https://github.com/prestojs/prestojs/pull/30/files#diff-ac3463492dddcc2cdcda76fde5083514L38) we you didn't even name it.

Can you confirm that this is consistent with how this has been named elsewhere? Are there any other project you know of (esp in React itself) that have a convention for how to name this?

Reply:
I couldn't find any naming convention externally - almost always destructured. It's called `currentStatePair` in the pagination branch but that's in the context of paginator - here there's no such context so either `paginationStatePair` or `currentPaginationStatePair`

Type definitions in React source are an unnamed tuple and I couldn't find any examples of naming it without destructuring